index.html

```diff
@@ -87,7 +87,13 @@ <h1 class="title">2048</h1>
     <p class="credits">
       Star Wars icons by <a href="http://sensibleworld.com/news/more-free-star-wars-icons/" title="Free Star Wars Icons by Sensible World">Jory Raphael</a>.
     </p>
-    <p class="repo-link"><small>hosted on <a href="https://github.com/0x0800/">GitHub</a></small></p>
+	  <hr>
+	  <p class="credits">
+		  Created by <a href="https://github.com/petrosh">Alex Petrosh</a>  <br>
+		  along with thanks to <a href="https://github.com/s10wen">Simon Owen
+		  </a> and <a href="https://github.com/card100">Henry Gruett</a>
+	  </p>
+    <p class="repo-link"><small>view source on <a href="https://github.com/0x0800/2048-STARWARS">GitHub</a></small></p>
```

Comment:
`credits` class is used by localization and conflict with icons credits, we need a new class and a new entry in  [js/localization.js](https://github.com/0x0800/2048-STARWARS/blob/gh-pages/js/localization.js).